bin/pint

```diff
@@ -113,7 +125,7 @@ for csp in cloud_providers:
 
 image_state = None
 image_states_data = ifsrequest.get_image_states_data(
-    None, None, 'json', None
+    None, None, result_format='json'
```

Comment:
Also changes the behavior, previously `get_image_states_data()` was effectively called with `region=None` now, because of the default setting this becomes `region=all`. I would think this change would break things.